modules/xresources/hm.nix

```
@@ -7,7 +7,7 @@
   config = lib.mkIf (config.stylix.enable && config.stylix.targets.xresources.enable) {
     xresources.properties = with config.lib.stylix.colors.withHashtag; with config.stylix.fonts; {
       "*.faceName" = monospace.name;
-      "*.faceSize" = sizes.terminal;
+      "*.faceSize" = builtins.toString sizes.terminal;
```

Consider simplifying:

```suggestion
      "*.faceSize" = toString sizes.terminal;
```

---

Done.